src/Nethermind/Nethermind.JsonRpc.Test/Modules/SubscribeModuleTests.cs

```diff
@@ -1093,5 +1093,18 @@ public void LogsSubscription_can_send_logs_with_removed_txs_when_inserted()
             var expectedResult = string.Concat("{\"jsonrpc\":\"2.0\",\"method\":\"eth_subscription\",\"params\":{\"subscription\":\"", logsSubscription.Id, "\",\"result\":{\"address\":\"0xb7705ae4c6f81b66cdb323c65f4e8133690fc099\",\"blockNumber\":\"0xd903\",\"data\":\"0x010203\",\"logIndex\":\"0x0\",\"removed\":true,\"topics\":[\"0x03783fac2efed8fbc9ad443e592ee30e61d65f471140c10ca155e937b435b760\"],\"transactionIndex\":\"0x0\",\"transactionLogIndex\":\"0x0\"}}}");
             expectedResult.Should().Be(serialized);
         }
+        
+        
+        
+        [Test]
+        public void CreateCustomSubscriptionViaDelegate_creating_result()
+        {
+            //Step 1 of test: registering a new subscription type.
+            
+            
+            string serialized = RpcTest.TestSerializedRequest(_subscribeRpcModule, "eth_subscribe", "newPendingTransactions");
+            var expectedResult = string.Concat("{\"jsonrpc\":\"2.0\",\"result\":\"", serialized.Substring(serialized.Length - 44,34), "\",\"id\":67}");
+            expectedResult.Should().Be(serialized);
+        }
```

Comment:
Is this test meaningful?

Reply:
Whoops! This was a loose end I forgot to delete. It has been corrected.